### ISSUE TITLE
add virtual Keyboard to browser

### ISSUE THREAD

**Issue by bhaggenmiller** [WebReflection/archibold.io#28]
Hey I am using your gjs browser script for a application in Kiosk mode. Unfortunately I am unable to add a virtualKeyboard. Do you have a recommendation for best way to implement this feature ? 

Thanks a lot

**Comment by WebReflection** [comment]
if you are on Weston that should come up automatically

**Comment by bhaggenmiller** [comment]
Ok thanks. is there a fast solution for x11?

**Comment by WebReflection** [comment]
yes, install weston instead 😁 ... I wouldn't know, sorry, I also use WPEWebKit these days

**Comment by bhaggenmiller** [comment]
Ok i did. What is the best way to start the application as systemd service ?

**Comment by WebReflection** [comment]
weston changed a lot these days ... there is some attempt in this repo but I am not sure it still works ... you want target network and graphics first then you should be able to start it **or** you use LightDM or GDM and let these handle the rest. https://github.com/canonical/lightdm

**Comment by WebReflection** [comment]
hah, this might do: https://wayland.pages.freedesktop.org/weston/toc/running-weston.html#running-weston-from-a-systemd-service
